Add a user guide section and FAQ (#18)
iiasa/ece-docs: user-guide.rst

Frequently Asked Questions
--------------------------

> .. rubric:: Why are there different *Scenario Explorer* instances for specific projects?

Reviewer: The word _instance_ might be too technical here, I'd just use the plural, Scenario Explorers:

```suggestion
.. rubric:: Why are there different Scenario Explorers for specific projects?
```

> Each project usually brings together different consortium partners and uses a

Reviewer: ```suggestion
Each project brings together different consortium partners, uses a
```

> Each project usually brings together different consortium partners and uses a

Reviewer: I'd include the points that each project covers different research questions and therefore produces different modeling results.

> project-specific list of variable and regions (see :ref:`project-templates`).

Reviewer: ```suggestion
project-specific list of variable and regions (see :ref:`project-templates`), tackles different research questions and therefore produces different results.
```

It is therefore easier to keep projects separate, with its own database **platform**,
*Scenario Explorer* instance and related GitHub repository (see :ref:`scenario-processing`).

> .. rubric:: What is the level of spatial granularity of scenarios?

Reviewer: In the effort of keeping the language as simple as possible;

```suggestion
.. rubric:: What is the level of spatial granularity of the data?
```

> 

Reviewer: I'd start with something along the lines of:
The Scenario Explorer as a tool is agnostic to the spatial resolution of the data.

Author: Following your advice to make the text less technical (instead of adding the word "agnostic").

The spatial granularity of the scenario data depends on the scope of the project and the
contributing modeling teams. The IAMC data format supports both aggregated data as well
as more detailed regional data like country-level resolution (see :ref:`countries`)
or the :ref:`nuts` used by the European Union.

.. rubric:: Which sectors are covered by the variable template?

The sectoral coverage of the data format is flexible and can be adapted to the needs of
the project. Read more about the :ref:`variables`.

> You can also look at the `common-definitions`_ repository for a list of variables

Reviewer: Since this is aimed at beginner users, linking to the GitHub directory without much further explanation might be confusing. 
I'd suggest to (also) link to the Excel version of common-definitions.

Author: The xlsx link is highlight in the common-definitions Readme, and I'd rather users are a bit confused when looking at a GitHub repository than directly downloading an xlsx file without knowing that it is coming from a larger ecosystem.
